### PR TITLE
add toucheism and touche.md

### PR DESCRIPTION
**Pull request by toucheism**

### CODE REVIEW

**Comment by toucheism** [comment]
Toucheism pull request

**Comment by toucheism** [comment]
Please review this, @casey-collab.

File: english/toucheism/touche.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+In life we all crave attention and some put it in one word "love,"
```
Comment (casey-collab) [inline english/toucheism/touche.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/toucheism/touche.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Like the feeling you get watching a slow roasting marshmellow; thinking 
+of it melting in your mouth. 
+
+A little bit of touche, would do a lot of good for our world.
+
+Let's all aim to touch a life positively at every contact. 
+
+Now how are you going to do that?
+
+with [food](../marshmellow.md) that cure the soul?
```
Comment (casey-collab) [inline english/toucheism/touche.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/toucheism/touche.md (hunk 1 above)
Comment (casey-collab) [inline english/toucheism/touche.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
It looks like the file `english/toucheism/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by toucheism** [comment]
Please review this, @casey-collab.

File: english/toucheism/toucheism.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Like the feeling you get watching a slow roasting marshmellow; thinking 
+of it melting in your mouth. 
+
+A little bit of touche, would do a lot of good for our world.
+
+Let's all aim to touch a life positively at every contact. 
+
+Now how are you going to do that?
+
+with [food](../marshmellow.md) that cure the soul?
```
Comment (casey-collab) [inline english/toucheism/toucheism.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/toucheism/toucheism.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+In life we all crave attention and some put it in one word "love,"
```
Comment (casey-collab) [inline english/toucheism/toucheism.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by toucheism** [comment]
Please review this, @casey-collab.

File: english/toucheism/toucheism.md (hunk 3 above)
Comment (casey-collab) [inline english/toucheism/toucheism.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by toucheism** [comment]
Please review this, @casey-collab.

File: english/toucheism/touche.md (hunk 1 above)
Comment (casey-collab) [inline english/toucheism/touche.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/toucheism/toucheism.md (hunk 4 above)
Comment (casey-collab) [inline english/toucheism/toucheism.md]: This file has also been added to a directory that already contained a page.

File: english/toucheism/toucheism.md (hunk 3 above)
Comment (casey-collab) [inline english/toucheism/toucheism.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/toucheism/touche.md (hunk 1 above)
Comment (casey-collab) [inline english/toucheism/touche.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
It looks like you've added two files that are very similar - `toucheism.md` and `touche.md`. You should use `git rm` to remove one of them, and I believe that should fix most of your problems. Then you'll need to make sure you've applied all your fixes in the one you keep (right now one of them has a correct link, but the other has a typo), and make sure the link in `marshmallow.md` leads to the one you keep.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**